tests/backbeat/Using.md

```diff
@@ -100,10 +100,17 @@
    and `.secrets.env`):
 
     ```
+    export CLOUDSERVER_HOST=<zenko-cloudserver-name>
```

Comment:
👍

Reply:
ya i noticed we use both ENDPOINT and HOST in setup :O